### PR TITLE
[circle-part-eval] Introduce Sqrt_Rsqrt part files

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce Sqrt_Rsqrt and Sqrt_Rsqrt_Add partition files to
test differenct cases of partitioning.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #6230, from draft #6231

**Comment by seanshpark** [comment]
https://github.com/Samsung/ONE/issues/6230#issuecomment-801618150 and below shows the graph for the partitions

**Comment by seanshpark** [comment]
@nnfw-bot test nncc-debug

File: compiler/circle-part-eval/parts/Part_Sqrt_Rsqrt_Add_003.part [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[partition]
+backends=cpu,acl_cl
+default=cpu
+comply=opcode
+
+[OPCODE]
+WWW=acl_cl
```
Comment (mhs4670go) [inline compiler/circle-part-eval/parts/Part_Sqrt_Rsqrt_Add_003.part:7]: For negative test?

Comment (seanshpark) [reply compiler/circle-part-eval/parts/Part_Sqrt_Rsqrt_Add_003.part:7]: Yes, something like that :)

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**